packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -326,12 +326,10 @@ export class DataList extends React.Component<DataListProps, DataListState> {
             isCompact && styles.modifiers.compact,
             gridBreakpointClasses[gridBreakpoint],
             wrapModifier && styles.modifiers[wrapModifier],
+            dragging && styles.modifiers.dragOver,
```

Comment:
This looks good to replace the existing behavior, but I think we need to change it so that `styles.modifiers.dragOver` is only applied while dragging, meaning it's removed once the item is dropped.

Reply:
It looks like the `dragging` state needs to be updated in the drag end handler, it is removed properly on a drag cancel. It should work with that updated.

Reply:
As @kmcfaul mentioned. `dragging` is a part of [the `DataList` state](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/DataList/DataList.tsx#L55) which is being updated to true in [`dragStart0`](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/DataList/DataList.tsx#L143-L146) and false in [`onDragCancel`](https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/DataList/DataList.tsx#L162-L166). These two are being called accordingly to the events when the user clicks and holds the element or releases the element.

Reply:
It should also be updated in `dragEnd0` so the style gets removed when the drag is successfully completed.